### ISSUE TITLE
parsing structures in c header file

### ISSUE THREAD

**Issue by jyothischali** [loarabia/Clang-tutorial#27]
Hi,
in the tutorial, https://github.com/loarabia/Clang-tutorial/wiki/TutorialOrig, it is explaine about parsing c file. But there are no enough information of parsing a c header file, getting the structure name, its member names & type, ..etc. Would that be added in the tutorial or is there another tutorial for that ?

**Comment by loarabia** [comment]
You're correct. There isn't a tutorial that shows off the explicit techniques for parsing a header. I'd love to see a pull request for such an example though.

**Comment by jyothischali** [comment]
Hi,
who can make a pull request for that ?

**Comment by loarabia** [comment]
I'm open to receiving pull requests from anyone. So for instance, if you would like to try to make the change and submit a pull request, I'd be happy to code-review and merge the change into the codebase for the benefit of others.
